Unreal/SystemLocalized/det/UnrealIntegrity.det

```diff
@@ -1,7 +1,7 @@
 ﻿[Public]
-Preferences=(Caption="Unreal Integrity",Parent="Networking")
-Preferences=(Caption="Integrity Server",Parent="Unreal Integrity",Class=UnrealIntegrity.IntegrityServer,Immediate=True)
-Preferences=(Caption="Whitelist",Parent="Unreal Integrity",Class=UnrealIntegrity.ExternalWhiteList,Immediate=True)
+Preferences=(Caption="Unreal Integrity",Parent="Networking",LangCaption="Integrität von Unreal")
```

Comment:
```suggestion
Preferences=(Caption="Unreal Integrity",Parent="Networking",LangCaption="Unreal Integrity")
```

"Unreal Integrity" is the given name of the feature, it is not some common concept that needs translation.